RGLServerPlugin/include/RGLServerPluginInstance.hh

```diff
@@ -79,14 +82,21 @@ private:
     std::vector<rgl_mat3x4f> lidarPattern;
     std::vector<rgl_vec3f> resultPointCloud;
     std::vector<float> resultIntensities;
+    std::vector<float> resultDistances;
 
     bool updateOnPausedSim = false;
+    bool publish_laserscan = false;
```

Comment:
They seem to use camel case here, so this should be like `publishLaserscan`

If we keep consistent with the naming convention of the code we're modifying, it makes it easier to get our changes accepted upstream (if we want to!).

Reply:
Thanks for spotting!

I'm not sure we will be contributing this to upstream but I have fixed it to be consistent.